src/groundlight/client.py

```diff
@@ -43,6 +43,7 @@ class Groundlight:
     """
 
     DEFAULT_WAIT: float = 30.0
+    DEFAULT_PATIENCE: float = 30.0
```

Comment:
At first blush I'd think we shouldn't need this in the SDK.  Detectors have a patience time configured in the cloud.  If customers don't specify patience here, it should always use whatever's in the cloud.